client/src/components/common/SmallTileDetails.tsx

```diff
@@ -28,6 +29,10 @@ export const SmallTileDetails: React.FC<{
   footer,
   textColor = "var(--mi-normal-foreground-color)",
 }) => {
+  const SubtitleComponent = () => <span>{subtitle}</span>;
```

Comment:
I'd prefer these in the view itself, I think it's a little confusing when we do that kind of stuff outside of the view.